normandy/recipes/tests/test_serializers.py

```diff
@@ -36,3 +37,55 @@ def test_it_uses_cdn_url(self, rf, settings):
         action = ActionFactory()
         serializer = ActionSerializer(action, context={'request': rf.get('/')})
         assert serializer.data['implementation_url'].startswith(settings.CDN_URL)
+
+
+@pytest.mark.django_db()
+class TestSignedRecipeSerializer:
+    def test_it_works_with_signature(self, rf):
+        recipe = RecipeFactory(signed=True)
+        action = recipe.action
+        serializer = SignedRecipeSerializer(instance=recipe, context={'request': rf.get('/')})
+
+        assert serializer.data == {
+            'signature': {
+                'signature': 'fake signature',
+                'timestamp': Whatever(),
+                'x5u': Whatever(),
+                'public_key': Whatever(),
+            },
+            'recipe': {
```

Comment:
Is it worth specifying this explicitly vs. just calling `RecipeSerializer` here?